### PR TITLE
Import and store legacy project counts by workflow

### LINKED ISSUE #1103: Legacy project activity counts should respect individual workflows
Linked to https://github.com/zooniverse/Panoptes-Front-End/issues/1028#issuecomment-121093777

Live event counts are per worklow so legacy workflow sums can be lost by classifying on legacy projects that only have the live workflow count. Update the activity count to be a json field and re-migrate the per workflow data. In turn update the events controller to update the per workflow counts.

### PR DESCRIPTION
**Pull request by edpaget**
Closes #1103 #1247

### CODE REVIEW

File: app/controllers/api/events_controller.rb [hunk 1]
```diff
@@ -2,8 +2,6 @@ module Api
 
   class EventsController < ApplicationController
 
-    skip_before_action :verify_authenticity_token, if: :json_request?
```
Comment (camallen) [inline app/controllers/api/events_controller.rb]: i assume that since we don't raise auth token error anymore this won't be a problem.

Comment (edpaget) [inline app/controllers/api/events_controller.rb]: Yup it should just have a null session.

**Comment by camallen** [comment]
pretty sure this is good. Do you want to mark the [migrated ones](https://github.com/camallen/Panoptes/blob/4c5cba47fa25b307112cb7074f946eca15b4d701/lib/zooniverse_user_subscription.rb#L83) in case you have to restart the migrations?

**Comment by edpaget** [comment]
Not really I'm going to run on Jenkins so if something interrupts the run I assume it's something that'll need to be fix globally.

**Comment by camallen** [comment]
Cool - merging.

**Pull request merged**